views/default/admin_domains_configuration.pdt

```diff
@@ -131,6 +132,18 @@ if (!(isset($is_ajax) ? $is_ajax : false)) {
                     </li>
                 </ul>
             </div>
+
+            <div class="title_row"><h3><?php $this->_('AdminDomains.configuration.heading_taxes');?></h3></div>
```

Comment:
Let's put this above the package options, I think it'll be used more often.

Reply:
Updated